core/lib/libtomcrypt/src/pk/ecc/sub.mk

```diff
@@ -0,0 +1,22 @@
+srcs-y += ecc.c
+srcs-y += ecc_ansi_x963_export.c
+srcs-y += ecc_encrypt_key.c
+srcs-y += ecc_free.c
+srcs-y += ecc_import.c
+srcs-y += ecc_shared_secret.c
+srcs-y += ecc_sizes.c
+srcs-y += ecc_verify_hash.c
+srcs-y += ltc_ecc_map.c
+srcs-y += ltc_ecc_mulmod.c
+srcs-y += ltc_ecc_points.c
+srcs-y += ltc_ecc_projective_dbl_point.c
+srcs-y += ecc_ansi_x963_import.c
+srcs-y += ecc_decrypt_key.c
+srcs-y += ecc_export.c
+srcs-y += ecc_get_size.c
+srcs-y += ecc_make_key.c
+srcs-y += ecc_sign_hash.c
+srcs-y += ltc_ecc_is_valid_idx.c
+srcs-y += ltc_ecc_mul2add.c
+srcs-y += ltc_ecc_mulmod_timing.c
+srcs-y += ltc_ecc_projective_add_point.c
```

Comment:
I don't think all the files are needed. Only the code that is needed by OP-TEE to implement the ECC crypto API should be included IMO.
Also, I suggest to order the files in alphabetical order.